docs/docs/reference-glossary.md

```diff
@@ -0,0 +1,100 @@
+---
+
+id: glossary
+title: A Glossary of Terms in React
+permalink: docs/glossary.html
+
+---
+
+#A Glossary of Terms in React
+
+
+## Single Page Application
+A single page application is an application that loads a single HTML page and all the necessary assets (CSS, Images & JavaScript) required for the application to run. Any interactions with the page or subsequent pages do not require a round trip to the server which means the page is not reloaded.
+
+
+## ES6/ES2015
+ ES6 AKA ES2015 is the current version of the ECMAScript Language Specification standard, which the JavaScript language is an implementation of. It includes many additions to the previous versions such as: arrow functions, classes, template literals, and `let` and `const` statements. 
+
+## Compilers/Transpilers
+A JavaScript compiler (or transpiler) takes JavaScript code, transforms it and returns JavaScript code in a different format. The most common use case is to take ES2015/ES6 syntax and transform it into syntax that browsers are capable of interpreting. Babel is the compiler used with React. 
```

Comment:
Ok yeah -- I'm still a little fuzzy on the difference between transpilers and compilers -- some of the articles I read about it said the words were interchangeable.